app/src/main/java/org/oppia/app/parser/StringToRatioParser.kt

```diff
@@ -78,3 +79,13 @@ class StringToRatioParser {
 private fun String.removeWhitespace(): String {
   return this.replace(" ", "")
 }
+
+/**
+ * Returns this Ratio in string format.
+ * E.g. [1, 2, 3] will yield to 1 to 2 to 3
+ */
+fun RatioExpression.toAccessibleAnswerString(context: Context): String {
```

Comment:
Doesn’t really seem like a parser function to me. I suggest moving this and tests to a dedicated ratio extensions file.